src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -44,28 +32,60 @@ const propTypes = {
 
 const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
+
+    emojiReactions: {},
 };
 
-const ReportActionItemReactions = (props) => {
+const ReportActionItemEmojiReactions = (props) => {
     const popoverReactionListAnchor = useRef(null);
-    const reactionsWithCount = _.filter(props.reactions, (reaction) => reaction.users.length > 0);
+    let totalReactionCount = 0;
 
+    // Each emoji is sorted by the oldest timestamp of user reactions so that they will always appear in the same order for everyone
+    const sortedReactions = _.sortBy(props.emojiReactions, (emojiReaction, emojiName) => {
+        // Since the emojiName is only stored as the object key, when _.sortBy() runs, the object is converted to an array and the
+        // keys are lost. To keep from losing the emojiName, it's copied to the emojiReaction object.
+        // eslint-disable-next-line no-param-reassign
+        emojiReaction.emojiName = emojiName;
+        return (
+            _.chain(emojiReaction.users)
+                .reduce((allTimestampsArray, userData) => {
+                    if (!userData) {
+                        return allTimestampsArray;
+                    }
+                    _.each(userData.skinTones, (createdAt) => {
+                        allTimestampsArray.push(createdAt);
+                    });
+                    return allTimestampsArray;
+                }, [])
+                .sort()
+                .first()
+                // Just in case two emojis have the same timestamp, also combine the timestamp with the
+                // emojiName so that the order will always be the same. Without this, the order can be pretty random
+                // and shift around a little bit.
+                .value() + emojiName
```

Comment:
If we introduced millisecond precision to the created timestamp for emoji reactions, I think it would solve this issue?

Reply:
I think adding ms precision makes it more unlikely, but does not solve this once and for all. This method will solve it for sure though.